pkg/util/validation/limits.go

```diff
@@ -38,6 +38,14 @@ func (e LimitError) Error() string {
 	return string(e)
 }
 
+type DisabledRuleGroup struct {
+	Namespace string `yaml:"namespace"`
+	Name      string `yaml:"name"`
+	User      string `yaml:"user,omitempty"`
```

Comment:
For user should it be `yaml:"-"`? I thought we don't want to have this field in yaml at all

Reply:
Done